mysqlcluster/syncer/variables.go

```diff
@@ -17,86 +17,92 @@ limitations under the License.
 package syncer
 
 import (
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
 	"github.com/radondb/radondb-mysql-kubernetes/utils"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 // log is for logging in this package.
 var log = logf.Log.WithName("mysqlcluster.syncer")
 
 // mysqlSysConfigs is the map of mysql system configs.
 var mysqlSysConfigs = map[string]string{
-	"default-time-zone":                  "+08:00",
-	"slow_query_log_file":                "/var/log/mysql/mysql-slow.log",
-	"read_only":                          "ON",
-	"binlog_format":                      "row",
+	"default-time-zone":   "+08:00",
+	"slow_query_log_file": "/var/log/mysql/mysql-slow.log",
+	"read_only":           "ON",
+	"binlog_format":       "row",
+	"log-bin":             "/var/lib/mysql/mysql-bin",
+	"log-timestamps":      "SYSTEM",
+	"innodb_open_files":   "655360",
+	"open_files_limit":    "655360",
+
+	"gtid-mode":                 "ON",
+	"enforce-gtid-consistency":  "ON",
+	"slave_parallel_type":       "LOGICAL_CLOCK",
+	"relay_log":                 "/var/lib/mysql/mysql-relay-bin",
+	"relay_log_index":           "/var/lib/mysql/mysql-relay-bin.index",
+	"master_info_repository":    "TABLE",
+	"relay_log_info_repository": "TABLE",
+	"slow_query_log":            "1",
+	"tmp_table_size":            "32M",
+	"tmpdir":                    "/var/lib/mysql",
+}
+
+var mysqlSpecialConfig = map[string]string{
```

Comment:
mysqlSpecialConfig -> mysql8Configs is better?
[ref](https://github.com/bitpoke/mysql-operator/blob/5a9fd7692c5deb8e264f65df57738c2afc9e694c/pkg/controller/mysqlcluster/internal/syncer/config_map.go#L102)

Reply:
this configs use both by mysql5.7 and mysql8, but mysql8 should deal it in special way.